wolfssh/test.h

```diff
@@ -546,7 +546,7 @@ static INLINE void build_addr(SOCKADDR_IN_T* addr, const char* peer,
 #endif
 
 
-static INLINE void tcp_socket(WS_SOCKET_T* sockFd)
+static INLINE void tcp_socket(WS_SOCKET_T* sockFd, int specifyProtocol, int targetProtocol)
```

Comment:
Can we make this a little smarter? Maybe have tcp_socket() and tcp6_socket() that call a utility function?